themes/digital.gov/src/scss/new/guides/_guide-page-head.scss

```diff
@@ -1,42 +1,22 @@
 @use "uswds-core" as *;
 
 .page-head-guides {
-  @include u-margin-bottom(8);
-
-  // Kicker
-  .kicker {
-    @include u-margin-bottom("05");
-    @include u-font("sans", "sm");
-    @include u-text("uppercase");
-    @include u-text("medium");
-    @include u-text("primary-vivid");
-
-    a {
-      @include u-text("no-underline");
-
-      &:hover {
-        @include u-border-bottom("2px", "primary-vivid", "solid");
-      }
-    }
-  }
+  @include u-margin-bottom(5);
 
   // Page Title
   h1 {
-    @include u-margin-top(0);
-    @include u-margin-bottom(2);
-    @include u-font("sans", "3xl");
-    @include u-text("thin");
+    @include u-font("sans", "2xl");
+    @include u-line-height("sans", 1);
+    @include u-text("heavy");
     @include u-text("primary-darker");
   }
+}
 
-  // Deck
-  .usa-intro {
-    @include u-margin(0);
-    @include u-margin-top("05");
-    @include u-margin-bottom(5);
-    @include u-text("light");
-    @include u-text("primary-darker");
-    @include u-font("sans", "lg");
-    @include u-line-height("sans", 3);
-  }
+.read-time {
+  @include u-font("sans", "2xs");
+  background-color: color("indigo-10");
+  border-radius: units(2);
+  display: inline-block;
+  margin: 0;
+  padding: units("05") units("105");
 }
```

Comment:
Let's move `read time` to it's own partial in `/src/scss/new` since we don't have a global typography partial.